### PR TITLE
Upgrade Kafka version

### PR DESCRIPTION
**Pull request by wai-kuen-shopkeep**
Upgrade to Kafka version 2.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
+                                         supervisor wget default-jre apt-transport-https
 
 # install kafka using .deb packages provided by Confluent
-RUN wget -qO - http://packages.confluent.io/deb/3.0/archive.key | apt-key add -
-RUN add-apt-repository "deb [arch=amd64] http://packages.confluent.io/deb/3.0 stable main"
+RUN wget -qO - https://packages.confluent.io/deb/5.0/archive.key | apt-key add -
+RUN add-apt-repository "deb [arch=amd64] https://packages.confluent.io/deb/5.0 stable main"
 RUN apt-get update && apt-get install -y confluent-platform-2.11
-RUN keytool -genkey -keyalg RSA -alias selfsigned -keystore /etc/kafka/kafka.jks -storepass password -keypass password -validity 360 -keysize 2048 -dname "CN=localhost,OU=foo,O=foo,L=bar,ST=bar,C=IT"
+RUN keytool -genkey -keyalg RSA -alias selfsigned -keystore /etc/kafka/kafka.jks -storepass password -keypass password -validity 360 -keysize 2048 -dname "CN=kafka,OU=foo,O=foo,L=bar,ST=bar,C=IT"
```
Comment (wai-kuen-shopkeep) [inline Dockerfile:11]: We need to change `CN=localhost` to `CN=kafka`.   Without it, we will have error verifying the elfsign certificate when kafka is launched.

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ kafka-docker
 
 [![Docker Automated Build](https://img.shields.io/docker/automated/paddycarey/kafka.svg)](https://hub.docker.com/r/paddycarey/kafka/)
 
-This repository provides a Dockerised Kafka instance (based on Confluent Platform 3.0.1/Scala 2.11) for development and testing use.
+This repository provides a Dockerised Kafka instance (based on Confluent Platform 3.0.1/Scala 2.11) for development and testing use.  This repository supports Kafka 2.0.1 which is a frok from this [repo](https://github.com/paddycarey/kafka-docker/).
```
Comment (smithtopher) [inline README.md]: frok

**Comment by paddycarey** [comment]
Does the README need updated also?

**Comment by wai-kuen-shopkeep** [comment]
The README has several references of `paddycarey/kafka` image.  We need to update that to `shopkeep/kafka`.  Also, we need to update the path like `kafka-console-producer.sh` to `kafka-console-producer` in several places in the README.

I assume the new repo will be public eventually.  If we are planing to make this new repo public, I think it is important for us to clarify things a little.  Especially the common name (CN) value which we need to specify correctly if user is not using 'kafka' as the hostname.

**Review by paddycarey — APPROVED** [review]

**Pull request merged**